onnxruntime/python/tools/transformers/models/whisper/whisper_helper.py

```diff
@@ -9,6 +9,7 @@
 from pathlib import Path
 from typing import Dict, Tuple, Union
 
+import datasets
```

Comment:
Can you move the below logic here? This was added because some CI pipelines do not have `datasets` installed.

https://github.com/microsoft/onnxruntime/blob/7b46b3155891cbf6c783e2535e413b92ae81b050/onnxruntime/python/tools/transformers/models/whisper/whisper_helper.py#L333-L342